lib/util.js

```diff
@@ -323,11 +323,19 @@ module.exports = {
               requestCount: 0,
               requests: [],
               children: {},
-              type: 'item-group'
+              type: 'item-group',
+              childrenCount: 0
             }));
           }
           // requestCount increment for the node we just added
           currentNode.children[currentPath[i]].requestCount += currentPathRequestCount;
+          // we are maintaining children count at every folder level
+          if (currentNode.name === '/') {
+            currentNode.childrenCount = pathLength;
```

Comment:
why?
if currentNode is `/`, it can still have multiple requests right? GET + POST, for example.